### PR TITLE
docs: add lambda endpoints usage documentation

### LINKED ISSUE #207: Add documentation how to launch API endpoint Lambda functions
Add documentation how to launch API endpoint Lambda functions

### PR DESCRIPTION
**Pull request by imincik**
closes #207

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+$ export AWS_ACCESS_KEY_ID=$(echo $CREDENTIALS | jq -r ".Credentials[\"AccessKeyId\"]")
+$ export AWS_SECRET_ACCESS_KEY=$(echo $CREDENTIALS | jq -r ".Credentials[\"SecretAccessKey\"]")
+$ export AWS_SESSION_TOKEN=$(echo $CREDENTIALS | jq -r ".Credentials[\"SessionToken\"]")
+```
+
+# Data Lake Lambda Endpoints Usage
+
+## Endpoint Request Format
+```
+$ export DATALAKE_LAMBDA_FUNCTION_ENDPOINT_NAME=<DATALAKE-LAMBDA-FUNCTION-ENDPOINT-NAME>
```
Comment (l0b0) [inline USAGE.md:25]: Do users need some instructions to retrieve the endpoint name? I'm not even completely sure I could get this command right.

**Review by l0b0 — CHANGES_REQUESTED** [review]

**Review by billgeo — CHANGES_REQUESTED** [review]
Thanks @imincik looks more or less fine to me. I think adding a bit more info about how to get the endpoints and account IDs etc and it's good to go. I will try and test this this week, but please merge this when it's ready.

File: USAGE.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Getting Access to Data Lake
+Currently, Data Lake allows read/write access for all users from all AWS accounts (users home
+accounts) specified in `DATALAKE_USERS_AWS_ACCOUNTS_IDS` during deployment time
+(see [README](README.md#aws-infrastructure-deployment-cdk-stack)).
+
+Temporary access credentials for Data Lake can be requested by running following commands while
+authenticated using user's home AWS account credentials:
+```
+export DATALAKE_AWS_ACCOUNT_ID=<DATALAKE-AWS-ACCOUNT-ID>
```
[the remark this answers is not in the document]
Comment (imincik) [reply USAGE.md:9]: I agree with Bill on this comment.

File: USAGE.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        --payload '{"httpMethod": "GET", "body": {"id": "cb8a197e649211eb955843c1de66417d", "type": "RASTER"}}' \
+        response.json && cat response.json
+
+    {"statusCode": 200, "body": {"created_at": "2021-02-01T13:38:40.776333+0000", "id": "cb8a197e649211eb955843c1de66417d", "owning_group": "A_XYZ_XYZ", "title": "Auckland 2020", "type": "RASTER", "updated_at": "2021-02-01T13:39:36.556583+0000"}}
+    ```
+
+* Example of Dataset delete request
+    ```
+    $ aws lambda invoke \
+        --function-name datasets-endpoint \
```
[the remark this answers is not in the document]
Comment (imincik) [reply USAGE.md:76]: For while yes, but we are talking about using AWS autogenerated resource names.

**Comment by imincik** [comment]
@billgeo , @l0b0 thanks for your comments. I applied changes to all of them and also add "Prerequisites" section listing all information need in order to start using DL.

I'll merge this PR now

**Pull request merged**